build.gradle

```diff
@@ -92,6 +92,22 @@ configure(allprojects - project(':distribution:archives:integ-test-zip')) {
   }
 }
 
+repositories {
```

Comment:
@cwperks this is not necessary:
 - the repositories are set https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/build.gradle#L96
 - the snapshots repositories are added by https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/src/main/java/org/opensearch/gradle/RepositoriesSetupPlugin.java

Reply:
I'm not sure how to fix this the right way, but the issue is that when dependabot is looking for outdated dependencies, its not looking at mavenCentral to get the latest version of a dependency.

Reply:
I'll take a look shortly as well, thanks!